module-presentation/src/main/kotlin/site/yourevents/memeber/facade/MemberFacade.kt

```diff
@@ -0,0 +1,84 @@
+package site.yourevents.memeber.facade
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import site.yourevents.guest.port.`in`.GuestUseCase
+import site.yourevents.invitation.domain.Invitation
+import site.yourevents.invitation.dto.response.InvitationInfoResponse
+import site.yourevents.invitation.port.`in`.InvitationUseCase
+import site.yourevents.invitationinformation.exception.InvitationInformationNotFoundException
+import site.yourevents.invitationinformation.port.`in`.InvitationInformationUseCase
+import site.yourevents.invitationthumnail.exception.InvitationThumbnailNotFoundException
+import site.yourevents.invitationthumnail.port.`in`.InvitationThumbnailUseCase
+import site.yourevents.member.domain.Member
+import site.yourevents.member.exception.MemberNotFountException
+import site.yourevents.member.port.`in`.MemberUseCase
+import site.yourevents.memeber.dto.response.MemberInfoResponse
+import site.yourevents.principal.AuthDetails
+import java.util.stream.Collectors
+
+@Service
+@Transactional
+class MemberFacade(
+    private val memberUseCase: MemberUseCase,
+    private val guestUseCase: GuestUseCase,
+    private val invitationUseCase: InvitationUseCase,
+    private val invitationInformationUseCase: InvitationInformationUseCase,
+    private val invitationThumbnailUseCase: InvitationThumbnailUseCase,
+) {
+    fun getMemberInfo(authDetails: AuthDetails): MemberInfoResponse {
+        val member = findMember(authDetails)
+
+        val receivedInvitationCount = guestUseCase.getReceivedInvitationCount(member)
+
+        val sentInvitationCount = invitationUseCase.countByMember(member)
+
+        return MemberInfoResponse.of(
+            member.getNickname(),
+            receivedInvitationCount,
+            sentInvitationCount
+        )
+    }
+
+    fun getSentInvitations(authDetails: AuthDetails): List<InvitationInfoResponse> {
+        val member = findMember(authDetails)
+
+        val sentInvitations = invitationUseCase.findByMember(member)
+
+        return mapInvitationsToResponse(sentInvitations)
+    }
+
+    fun getReceivedInvitations(authDetails: AuthDetails): List<InvitationInfoResponse> {
+        val member = findMember(authDetails)
+
+        val receivedInvitations = guestUseCase.getReceivedInvitations(member)
+
+        return mapInvitationsToResponse(receivedInvitations)
+    }
+
+    private fun findMember(authDetails: AuthDetails): Member {
+        val memberId = authDetails.uuid
+
+        return memberUseCase.findById(memberId) ?: throw MemberNotFountException()
+    }
+
+    private fun mapInvitationsToResponse(invitations: List<Invitation>): List<InvitationInfoResponse> {
+        return invitations.stream()
+            .map { invitation -> createInvitationInfoResponse(invitation) }
+            .collect(Collectors.toList())
```

Comment:
`toList()` 메서드를 통해 불변 컬렉션을 반환하도록 하는 것이 더욱 좋지 않을까요?
```suggestion
            .toList()
```

Reply:
오 그렇네요! 수정하겠습니다!